### ISSUE TITLE
[ENHANCEMENT] Disable other options if three categories are selected while creating a post

### ISSUE THREAD

**Issue by chinmaykunkikar** [krishnaacharyaa/wanderlust#66]
### Description

To improve the current UX, disable the remaining categories when the user selects three categories while a new blog post is being created.

### Current Behavior

Currently if a user selects more than three categories, they won't get any feedback until after clicking on the "Create Blog" button.
This degrades the UX as they will have to unselect certain categories and try to click on the submit button again.

### Proposed Enhancement

I propose that once the user selects three categories, other categories go into a disabled state.
If the user de-selects one of the three, all the categories will be shown as available again.

### Use Case

This enhancement benefits anyone who's trying to create a blog post.

**Comment by SanjeevKrSingh** [comment]
Hey @chinmaykunkikar @krishnaacharyaa  I love to fix this enhancement can you plz assign it to me. So that I start working on it.

**Comment by krishnaacharyaa** [comment]
Yes please @SanjeevKrSingh, just try to come up with most optimized way possible :)

**Comment by vivek6201** [comment]
Hey! @krishnaacharyaa , if anybody is not working on this issue can you assign this to me?

**Comment by krishnaacharyaa** [comment]
Sure @vivek6201, It's almost done in https://github.com/krishnaacharyaa/wanderlust/pull/73, just try to resolve the issues, that would suffice.

**Comment by vivek6201** [comment]
hey issue resolved, i am rasing a pr please check and merge. @krishnaacharyaa
